lightningd/channel/channel.c

```diff
@@ -830,10 +833,31 @@ static void handle_ping(struct peer *peer, const u8 *msg)
 			    &peer->channel_id,
 			    WIRE_CHANNEL_PEER_BAD_MESSAGE,
 			    "Bad ping");
+
+	status_trace("Got ping, sending %s", pong ?
+		     wire_type_name(fromwire_peektype(pong))
```

Comment:
Isn't checking `pong == NULL` easier than to attempt to `peektype` a potentially `NULL` reference?